app/routing/ArrivalsRouter.scala

```diff
@@ -80,6 +85,26 @@ class ArrivalsRouter @Inject() (
       )
   }
 
+  def getArrivalMessage(arrivalId: String, messageId: String): Action[Source[ByteString, _]] = route {
+    case Some(VERSION_2_ACCEPT_HEADER_PATTERN()) =>
```

Comment:
As you have removed formatter from controller so you need to  use case Some(VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE_JSON) => and remove the VERSION_2_ACCEPT_HEADER_PATTERN
@liandra-giacomine / @drnaylor-oc I hope this is what we have agreed?

Reply:
No, this is correct. We want developers to test using `application/vnd.hmrc.2.0+json-xml` now, because that won't change.

Reply:
but this is going to work for application/vnd.hmrc.2.0+json as well as application/vnd.hmrc.2.0+json+xml if I am not mistaken? so should not better to use VERSION_2_ACCEPT_HEADER_VALUE_JSON and remove the action provider from controller?

Reply:
Correct, it will, and this is intended. We need it to work for the `json-xml` version more so than the `json` variant anyway, what you're suggesting will mean it _won't_ work for that.